### PR TITLE
Aesthetics

### PR DESCRIPTION
**Pull request by andrewjbe**
There are only two changes:

- Added the hex sticker as the package logo with `usethis::use_logo()`
- Added a fun loading message for when you call `library(ojodb)`; if this is too much we can remove it lol

### CODE REVIEW

**Review by brancengregory — DISMISSED** [review]
Howdy neighbor, this here commit is looking mighty fine. Why don't y'all wander 'ore yonder to the main branch and make yourselves at home

**Comment by brancengregory** [comment]
Leaving this as a bookmark, will explain what's needed later
https://github.com/tidyverse/tidyverse/blob/8ec2e1ffb739da925952b779925bb806bba8ff99/R/zzz.R

**Comment by andrewjbe** [comment]
Adding the SQL scripts on this branch just because it was the one I was working on

**Comment by andrewjbe** [comment]
The checks are failing because our `ojo_list_schemas()` testthat test is failing; the order of the schemas it lists has changed for some reason, probably just a quirk from the adjustment in roles?

File: R/zzz.R [hunk 1]
```diff
@@ -1,40 +1,10 @@
...
 ojodbStartupMessage <- function() {
 
-  ojo_art <- r"{                   _           _ _
- ╔══    ══╗   ___ (_) ___   __| | |__
- ║/      \║  / _ \| |/ _ \ / _` | '_ \
-            | (_) | | (_) | (_| | |_) |
- ║\      /║  \___// |\___/ \__,_|_.__/
- ╚══    ══╝     |__/
-                          version }"
+  ojo_art <- "                   _           _ _\n \u2554\u2550\u2550    \u2550\u2550\u2557   ___ (_) ___   __| | |__\n \u2551/      \\\u2551  / _ \\| |/ _ \\ / _` | '_ \\\n            | (_) | | (_) | (_| | |_) |\n \u2551\\      /\u2551  \\___// |\\___/ \\__,_|_.__/\n \u255a\u2550\u2550    \u2550\u2550\u255d     |__/\n                          version "
```
Comment (andrewjbe) [inline R/zzz.R:3]: I rewrote this using the escapes instead of the non-ASCII characters so that the `devtools::check()` warning would go away

**Review by brancengregory — APPROVED** [review]

**Pull request merged**